### PR TITLE
Python Web Conference 2023 recap

### PR DESCRIPTION
**Pull request by jerivas**

### CODE REVIEW

**Comment by jerivas** [comment]
[Python Web Conference 2023 Recap](https://trello.com/c/k7WsKobm/110-python-web-conference-2023-recap)

**Review by jgerigmeyer — APPROVED** [review]

File: content/blog/2023/python-web-conference-2023.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+- [WebGoat](https://owasp.org/www-project-webgoat/): *a deliberately insecure
+  application that allows interested developers just like you to test
+  vulnerabilities commonly found in Java-based applications that use common and
+  popular open source components.*
+  [Python](https://owasp.org/www-project-pygoat/) and
+  [Node.js](https://owasp.org/www-project-node.js-goat/) versions are also
+  available.
+- [Juice Shop](https://owasp.org/www-project-juice-shop/): *the most modern and
+  sophisticated insecure web application! It can be used in security trainings,
+  awareness demos, CTFs and as a guinea pig for security tools!*
```
[the remark this answers is not in the document]
Comment (jerivas) [reply content/blog/2023/python-web-conference-2023.md]: I think you're right. I copied these descriptions from the official site but I think it's worth explaining it. Fixed in 8a232dcd6c900e044a7970e7d3096facc3d144a5

File: content/blog/2023/python-web-conference-2023.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  fixing vulnerabilities.*
+
+## 1-click deploys of Python web apps to Azure
+
+By [Pamela
+Fox](https://2023.pythonwebconf.com/presentations/1-click-deploys-of-python-web-apps-to-azure).
+
+I'm constantly on the lookout for new platforms to deploy my Python web apps to.
+I'm specially interested in platforms that allow the team to automate as much as
+possible, use an "infrastructure as code" pattern, and have security and
```
[the remark this answers is not in the document]
Comment (jerivas) [reply content/blog/2023/python-web-conference-2023.md]: Fixed in8a232dcd6c900e044a7970e7d3096facc3d144a5

**Review by SondraE — APPROVED** [review]
A fantastic article! I have two suggestions.

**Intro Paragraph**:
I would add a little more to the first section to entice people to read the article even if they haven't heard of PWC. Maybe something like what I wrote below, but it could be anything you think would make people curious.
"OddBird sponsored Python Web Conference 2023 and sent me to attend. In this article I showcase my favorite talks and activities from this excellent online event, including resources to assess the security of your applications and websites, an intro to Azure, and a competition to discover whether Django, Flask, or FastAPI is better."

**Titles**
It looks like you followed the capitalization that each person used for their talk. I can see an argument for that, but I think I would recommend following OddSite's style guide for more visual consistency within the article.
https://www.oddbird.net/styleguide/writing
Use Title Case: Capitalize the first letter of every word except articles (the, a, an), prepositions (to, of, with) , and conjunctions (to, and).

**Comment by jerivas** [comment]
Thanks @SondraE! I just pushed a new round of changes incorporating your suggestions

File: content/blog/2023/python-web-conference-2023.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
+---
+title: Python Web Conference 2023 Recap
+author: ed
+date: 2023-03-21
```
Comment (jgerigmeyer) [inline content/blog/2023/python-web-conference-2023.md:4]:
```suggestion
date: 2023-03-22
```

**Review by jgerigmeyer — APPROVED** [review]

**Pull request merged**